api/services/control/control.proto

```diff
@@ -138,7 +138,10 @@ message VertexLog {
 message VertexWarning {
 	string vertex = 1 [(gogoproto.customtype) = "github.com/opencontainers/go-digest.Digest", (gogoproto.nullable) = false];
 	int64 level = 2;
```

Comment:
What's the purpose of `level`? Is it a threshold controlling how severe a warning must be?

Reply:
Yes, it is currently unused though but is propagated to the client if the frontend sets it.